gap/PGMatrix.gd

```diff
@@ -0,0 +1,96 @@
+#!
+#!
+#! @Chapter Hyperbolic lattice symmetries
+
+#!
+#! @Section Point group matrices
+#! @SectionLabel PGMatrix
+#!
+#! Symmetry elements of triangle groups might act non-trivially on hyperbolic momenta. 
+#! As such, the degrees of freedom of observables and topological invariants, like the 
+#! first Chern numbers in momentum space, might be contstrained by the hyperbolic lattice 
+#! symmetries of underlying models. We can determine how hyperbolic momenta transform as 
+#! hyperbolic lattice symmetry transformations act on Abelian states. This can be extracted 
+#! by constructing point-group matrices.
+
+#! @Description
+#!   Construct point-group matrices for hyperbolic lattice <A>symmetries</A>, given as words in 
+#!   the triangle group <A>fulltgFp</A> ($\Delta$), a finitly presented group (see <Ref Sect="Section_TriangleGroups"/>), 
+#!   which is either a single symmetry or a list of symmetries for a unit cell <A>cell</A>, 
+#!   given as a `TGCell` object (see <Ref Sect="Section_TGCellGraph"/>).
+#!   The <A>quotient</A> used in the construction of the unit cell <A>cell</A> must be specified 
+#!   as well, which is either an integer giving the position in Conder’s list or a list of the 
+#!   form `[genus, number]` giving the genus and number in Conder’s list of the quotient. 
+#!   
+#!   The option `symNames` can be used to specify the names of the <A>symmetries</A>. If a 
+#!   single symmetry is provided in the argument <A>symmetries</A>, `symNames` takes a single
+#!   string, otherwise a list of strings with the same number of elements as in <A>symmetries</A>
+#!   must be provided. The names will be used to label the corresponding point-group matrices.
```

Comment:
Mention what happens by default (namely it using the symmetry elements as far as I understand).